From 4a49f8c5aecc01984961aae1e0bf58a648b18e80 Mon Sep 17 00:00:00 2001
From: Sourav1000888 <sourbhrawat3@gmail.com>
Date: Thu, 30 Apr 2026 22:41:02 +0530
Subject: [PATCH] file

---
 app.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/app.py b/app.py
index 1f96084..b021f2f 100644
--- a/app.py
+++ b/app.py
@@ -9,3 +9,6 @@ for i in range(20):
 
 def hello():
     return {'message'  : 'hello world'}
+
+def file():
+    return {'message' : 'hello world'}
\ No newline at end of file
-- 
2.47.0.windows.1

